interface/login/login.php

```diff
@@ -223,6 +223,8 @@ function getLanguagesList(): array
     $cookie = json_encode($cookie);
 }
 
+$loginlogolocation = dirname(__DIR__, 2) . "/sites/" . $_SESSION['site_id'] . "/images/login-logo.svg";
```

Comment:
btw,
This line is where would put the check for the custom logo file and the comment

Reply:
The global would need a space where the name of the file can be entered. Is that what you were thinking?

Reply:
Look at $GLOBALS['OE_SITES_BASE'] instead of dirname()